File: flip.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    int n,i,j,flip,temp;
    while(cin>>n){
        flip=0;
        vector<int> nums(n);
        for(i=0;i<n;i++){
            cin>>nums[i];
        }
        for(i=0;i<n;i++){
            for(j=0;j<n-i-1;j++){
                if(nums[j]>nums[j+1]){
                    temp = nums[j];
                    nums[j]=nums[j+1];
                    nums[j+1]=temp;
                    flip++;
                }
            }
        }
        cout<<"Minimum exchange operations : "<<flip<<endl;
    }
    return 0;
}

File: flip.py
import sys

def main():
    data = sys.stdin.read().strip()
    elements = data.split()
    start=0
    while True:
        n = int(elements[start])
        numbers = list(map(int, elements[start+1:start+n+1]))
        flips=0
        for i in range(len(numbers)-1):
            for j in range(len(numbers)-i-1):
                if numbers[j]>numbers[j+1]:
                    numbers[j], numbers[j+1] = numbers[j+1], numbers[j]
                    flips+=1
        print("Minimum exchange operations : "+str(flips))
        start+=n+1
        if start>len(elements)-1:
            break
    
if __name__ == "__main__":
    main()


        




        